inst/extdata/energy/IEA_product_fuel.csv

```diff
@@ -0,0 +1,74 @@
+# File: IEA_product_fuel.csv
+# Title: Mapping from products in IEA energy balances to intermediate fuel names
+# Units: NA
+# Source: International Energy Agency, 2012, Energy Balances of OECD Countries 1960-2010 and Energy Balances of Non-OECD Countries 1971-2010
+# Description: The conversion factors convert from the units in IEA Energy Balances (kilotonnes of oil equivalent, terajoules of heat, and gigawatt-hours of electricity) to exajoules, and all energy inputs are indicated as positive numbers. In the IEA Energy Balances, the standard adopted is that for energy production and transformation sectors, energy inputs are negative and energy outputs are positive, whereas for energy consumption sectors, energy inputs are positive. We don't use that convention in GCAM; all energy inputs (and outputs) are indicated as positive numbers.
```

Comment:
Sure--I'm at a meeting now and have a phone call in an hour. Is getting it done by COB today OK?